### PR TITLE
Add MTE-1059 [v116] Add New Bookmarks Perf Tests and DBs

### PR DESCRIPTION
**Pull request by jjSDET**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-1059)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/FXIOS-15484)

## :bulb: Description
Adding new performance tests and dbs for bookmarks:
* testPerfBookmarks1startUp
* testPerfBookmarks1openMenu
* testPerfBookmarks100startUp
* testPerfBookmarks100openMenu
* testBookmarksDatabase1-places.db
* testBookmarksDatabase100-places.db

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and ensured the tests suite is passing
- [ ] Implemented accessibility and tested on UI related work (minimum Dynamic Text and VoiceOver)
- [ ] Updated documentation / comments for complex code and public methods if needed

### CODE REVIEW

File: Tests/XCUITests/PerformanceTests.swift [hunk 1]
```diff
@@ -169,6 +175,70 @@ class PerformanceTests: BaseTestCase {
         }
     }
 
+    func testPerfBookmarks1startUp() {
+        waitForTabsButton()
+//        app.terminate()
```
Comment (isabelrios) [inline Tests/XCUITests/PerformanceTests.swift:180]: Nit: debug line not needed

File: Tests/XCUITests/PerformanceTests.swift [hunk 2]
```diff
@@ -169,6 +175,70 @@ class PerformanceTests: BaseTestCase {
         }
     }
 
+    func testPerfBookmarks1startUp() {
```
Comment (isabelrios) [inline Tests/XCUITests/PerformanceTests.swift:178]: Do we also want one test like this for History?

File: Tests/XCUITests/PerformanceTests.swift (hunk 1 above)
Comment (jjSDET) [reply Tests/XCUITests/PerformanceTests.swift:180]: whoops, left it in. good catch. removing...

File: Tests/XCUITests/PerformanceTests.swift (hunk 2 above)
Comment (isabelrios) [reply Tests/XCUITests/PerformanceTests.swift:178]: I see, it is in a different ticket and PR, sorry!

**Review by isabelrios — APPROVED** [review]

File: Tests/XCUITests/PerformanceTests.swift (hunk 2 above)
Comment (jjSDET) [reply Tests/XCUITests/PerformanceTests.swift:178]: no worries! i'm testing the new suite of tests now. just have to figure out the best way to get good performance numbers after populating the entire 1k and 4k cells in `History Panel`

**Pull request merged**